vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -613,6 +613,26 @@ supported:
         <Annotation Term="Core.Description" String="Supports [passing query options in the request body](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_PassingQueryOptionsintheRequestBody)" />
       </Term>
 
+      <Term Name="PayloadExpressionSupport" Type="Capabilities.PayloadExpressionSupportType" Nullable="false" AppliesTo="EntityContainer">
+        <Annotation Term="Core.Description" String="Whether expressions for parameters in action payloads [OData-JSON, section 18] and `/$query` payloads [OData-URL, section 4.17] are supported" />

```

Comment:
Section numbers are not stable, better reference the section title and make this a Markdown link by adding parens with the URL.